pangram/pangram.rb

```diff
@@ -0,0 +1,39 @@
+class Pangram
+  def self.pangram?(input)
+
+
+    if input==""
+      return false
+    end
+
+
+      input=input.downcase.split('')
+      flag=true
+
+
+      ('a'..'z').each do |i|
+
+          if flag==false
+            flag=false
+            break
+          end
+
+          input.each do |j|
+                if i==j
+                  flag=true
+                  break
+
+                else
+                  flag=false
+                end
+
+          end
+      end
+
+      if flag==true
```

Comment:
Is it really required to test flag is true then return true else false. Instead of that just return flag